tests/local-inputs-compressed/builds.yaml

```diff
@@ -0,0 +1,23 @@
+inputs:
+  - name: test-local-compressed-asia-sequences
+    metadata: tests/local-inputs-compressed/data/asia_metadata.tsv.xz
+    sequences: tests/local-inputs-compressed/data/asia_sequences.fasta.xz
+  - name: test-local-compressed-europe-aligned
+    metadata: tests/local-inputs-compressed/data/europe_metadata.tsv.xz
+    aligned: tests/local-inputs-compressed/data/europe_aligned.fasta.xz
+  - name: test-local-compressed-oceania-masked
+    metadata: tests/local-inputs-compressed/data/oceania_metadata.tsv.xz
+    masked: tests/local-inputs-compressed/data/oceania_masked.fasta.xz
+  - name: test-local-compressed-americas-filtered
+    metadata: tests/local-inputs-compressed/data/americas_metadata.tsv.xz
+    filtered: tests/local-inputs-compressed/data/americas_filtered.fasta.xz
+
+builds:
+  test-local-compressed:
+    subsampling_scheme: small
+
+subsampling:
+  small:
+    one-hundred:
```

Comment:
Minor point: we may not want to name the subsampling scheme after a potentially mutable parameter like max sequences. I know the name doesn't matter much, but it would be confusing to see `one-hundred` here after we eventually reduce the max sequences to 50 for faster CI runs. 😄

Reply:
Good call - changed to `small-sample`